docs/bigdata/README.md

```diff
@@ -0,0 +1,272 @@
+
+To know more information about these options please visit [https://www.cloudera.com/documentation/enterprise/5-11-x/topics/admin_hive_on_s3_tuning.html](https://www.cloudera.com/documentation/enterprise/5-11-x/topics/admin_hive_on_s3_tuning.html)
+
+
+![alt_text](images/image13.png "image_tooltip")
+
+
+Once the config changes are applied, proceed to restart all Hive services.
+
+
+![alt_text](images/image14.png "image_tooltip")
+
+
+## **4. Run Sample Applications**
+
+After installing Hive, Hadoop and Spark successfully, we can now proceed to run some sample applications to see if they are configured appropriately.  We can use Spark Pi and Spark WordCount programs to validate our Spark installation. We can also explore how to run Spark jobs from the command line and Spark shell.
+
+
+### **4.1 Spark Pi**
+
+Test the Spark installation by running the following compute-intensive example, which calculates pi by “throwing darts” at a circle. The program generates points in the unit square ((0,0) to (1,1)) and counts how many points fall within the unit circle within the square. The result approximates pi.
+
+Follow these steps to run the Spark Pi example:
+
+*   Login as user **‘spark’.**
+*   When the job runs, the library can now use **MinIO **during intermediate processing.
+*   Navigate to a node with Spark client and access the spark2-client directory:
+
+```
+cd /usr/hdp/current/spark2-client
+su spark
+```
+
+
+*   Run the Apache Spark Pi job in yarn-client mode, using code from **org.apache.spark**:
+
+```
+./bin/spark-submit --class org.apache.spark.examples.SparkPi \
+    --master yarn-client \
+    --num-executors 1 \
+    --driver-memory 512m \
+    --executor-memory 512m \
+    --executor-cores 1 \
+    examples/jars/spark-examples*.jar 10
+```
+
+
+
+The job should produce an output as shown below. Note the value of pi in the output.
+
+
+```
+17/03/22 23:21:10 INFO DAGScheduler: Job 0 finished: reduce at SparkPi.scala:38, took 1.302805 s
+Pi is roughly 3.1445191445191445
+```
+
+
+Job status can also be viewed in a browser by navigating to the YARN ResourceManager Web UI and clicking on job history server information.
+
+
+### **4.2 WordCount**
+
+WordCount is a simple program that counts how often a word occurs in a text file. The code builds a dataset of (String, Int) pairs called counts, and saves the dataset to a file.
+
+The following example submits WordCount code to the Scala shell: Select an input file for the Spark WordCount example. We can use any text file as input.
+
+
+
+*   Login as user **‘spark’.**
+*   When the job runs, the library can now use **MinIO **during intermediate processing.
+*   Navigate to a node with Spark client and access the spark2-client directory:
+
+```
+cd /usr/hdp/current/spark2-client
+su spark
+```
+
+
+
+The following example uses _log4j.properties_ as the input file:
+
+
+#### **4.2.1 Upload the input file to HDFS:**
+
+
+```
```

Comment:
Do we need to explicitly state that testbucket is within MinIO?

Reply:
We don't need to @eco-minio